deploy/bin/update-worker-types.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const bluebird = require('bluebird');
+const workerTypes = require('../etc/worker-types.json');
+const amis = require('../../docker-worker-amis.json');
+const jsonfile = bluebird.Promise.promisifyAll(require('jsonfile'));
+const utils = require('./worker_type_utils');
+
+/*
+ * Write worker-types configuration to a json file
+ */
+async function backupWorkerTypes(client) {
+  console.log('Creating worker-types backup file.');
+  const names = await client.listWorkerTypes();
+  const workerTypes = names.map(name => {
+    const w = new utils.WorkerType(client, name);
+    return w.workerType();
+  });
+
+  await Promise.all(workerTypes).then(
+    result => jsonfile.writeFileAsync(
```

Comment:
we only use jsonfile once here.  Is it worth using it and needing another module for this one use case?

Reply:
Also since we're only writing one file here, why not use writeFileSync and not need bluebird to promisfy it?

Reply:
bluebird is already present in docker-worker, the as `backupWorkerTypes` already returns a `Promise`, using an async version seems the right thing.